client/components/GridMosaic.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React from 'react';
+
+const GridMosaic = ({ photo, openModal }) => {
+  const items = [];
+
+  for (let i = 0; i < 5; i += 1) {
+    items.push(
+      <div className={`box${i}`}>
+        <img
+          key={`${i}value`}
+          className="image_mosaic"
```

Comment:
This applies throughout your whole div, but HTML usually uses kebab case, like this:
```suggestion
          className="image-mosaic"
```